openapi.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.0
+info: 
+  title: To-do API
+  description: To-do app api
+  version: '1.0'
+  
+paths: 
+  /to-dos:
+    get:
+      summary: Return list of to-do
+      responses:
+        '200':
+          description: A Json array of to-dos
+          content: 
+            application/json:
+              schema:
+                type: array
+                items:
+                  content: 
+                  $ref: '#/components/schemas/to-do'
+        '301':
```

Comment:
3'lü hata code larını buraya yazmana gerek yok onun dışında burada 404 dönebilirsin veya bir db hatası olduğunda 500 dönebiliriz. 3xx ler redirection ile falan alakalı

Reply:
bir kaç örnek istiyorum demiştin onun için ekledim değiştiririm

Reply:
aslında her bir response içerisinde senden 2xx lü 4xx lü ve 5xx lü hata kodları istiyorum